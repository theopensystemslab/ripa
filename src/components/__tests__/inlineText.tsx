import { cleanup, fireEvent, render, wait } from "@testing-library/react";
import * as React from "react";

import TextQuestion from "../InlineText";

afterEach(cleanup);

describe("Text Question Component", () => {
  it("should render snapshot", () => {
    const { asFragment } = render(<TextQuestion />);
    expect(asFragment()).toMatchSnapshot();
  });
});
