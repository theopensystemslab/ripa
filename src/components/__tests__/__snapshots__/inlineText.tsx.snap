// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Question Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    autocapitalize="off"
    autocorrect="off"
    class="MuiInputBase-root MuiInput-root makeStyles-root-1"
    spellcheck="false"
  >
    <input
      autocomplete="off"
      class="MuiInputBase-input MuiInput-input makeStyles-input-3"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
